tajo-catalog/tajo-catalog-server/src/test/java/org/apache/tajo/catalog/TestCatalog.java

```diff
@@ -64,18 +62,10 @@
 	
 	static CatalogServer server;
 	static CatalogService catalog;
+  static String testDir;
 
-  public static TajoConf newTajoConfForCatalogTest() throws IOException, UnsupportedCatalogStore {
-    return CatalogTestingUtil.configureCatalog(new TajoConf(), setupClusterTestBuildDir().getAbsolutePath());
-  }
-
-  public static File setupClusterTestBuildDir() throws IOException {
-    String randomStr = UUID.randomUUID().toString();
-    String dirStr = CommonTestingUtil.getTestDir(randomStr).toString();
-    File dir = new File(dirStr).getAbsoluteFile();
-    // Have it cleaned up on exit
-    dir.deleteOnExit();
-    return dir;
+  public static TajoConf newTajoConfForCatalogTest(String testDir) throws IOException, UnsupportedCatalogStore {
```

Comment:
It would be better if it returns a pair of TajoConf and auto-generated directory instead of taking ``testDir``, especially if it is used to make a temporary directory rather than a designated directory.